vnext/PropertySheets/ReactPackageDirectories.props

```diff
@@ -36,8 +36,6 @@
     <!-- Normalize the directory names -->
     <ReactNativeWindowsDir>$([MSBuild]::NormalizeDirectory($(ReactNativeWindowsDir)))</ReactNativeWindowsDir>
     <ReactNativePackageDir>$([MSBuild]::NormalizeDirectory($(ReactNativePackageDir)))</ReactNativePackageDir>
-    <ReactNativeDir>$([MSBuild]::NormalizeDirectory($(ReactNativeDir)))</ReactNativeDir>
```

Comment:
Should we remove the other normalization? From my understanding, the only reason they didn't have problems on your machine was that they were already absolute paths (so normalizing would he a no-op).

Reply:
That's possible... I dunno, I assume someone wrote things this way for a reason? I can try removing all of them, sure! ¯\_(ツ)_/¯

Reply:
Nope, turns out there are good reasons to normalize the other directories, otherwise you get nice errors like:
```
1>C:\Users\hpratt\source\repos\react-native-windows\vnext\Folly\Folly.vcxproj(273,5): warning : Folly required to build without microsoft/react-native. - Downloading folly...
1>C:\Users\hpratt\source\repos\react-native-windows\vnext\Folly\Folly.vcxproj(274,5): error : MSB3923: Failed to download file "https://github.com/facebook/folly/archive/v2019.09.30.00.zip".  Could not find a part of the path 'C:\Users\hpratt\source\repos\react-native-windows\\node_modules\react-native\..\..\node_modules\.folly\folly-2019.09.30.00..\.follyzip'.

6>C:\Users\hpratt\source\repos\react-native-windows\vnext\ReactWindowsCore\ReactWindowsCore.vcxproj(205,5): error : The value of ReactNativeWindowsDir should be the actual location of the package, not a symlink.
6>C:\Users\hpratt\source\repos\react-native-windows\vnext\ReactWindowsCore\ReactWindowsCore.vcxproj(205,5): error :      MSBuildThisFileDirectory: 'C:\Users\hpratt\source\repos\react-native-windows\vnext\ReactWindowsCore\' - ReactNativeWindowsDir: 'C:\Users\hpratt\source\repos\react-native-windows\\vnext\ReactWindowsCore\''
```

Reply:
Oh right, we're doing directory traversal for that one. I'm guessing it's still safe to not normalize the others, but up to you.